.eslintrc.json

```diff
@@ -5,6 +5,15 @@
     "browser": true,
     "node": true
   },
-  "extends": ["@guardian/eslint-config-typescript"],
-  "ignorePatterns": ["dist", "coverage"]
+  "parserOptions": {
```

Comment:
Not for this PR, but I wonder whether there's benefit of a CLI tool a la create-react-app that hides away the eslint? Not sure how I feel about it

Reply:
yeah i think it def worth looking into. i'm not sure how you get editor integration that way though?